From c12516ad62fe57ed816f21504a8de3b3fcb1d24a Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:42 -0400
Subject: [usb] core/hcd: if-else-braces fixed

Message-id: <1410887463-20674-31-git-send-email-dzickus@redhat.com>
Patchwork-id: 95575
O-Subject: [RHEL7 PATCH 030/591] drivers: usb: core: hcd: if-else-braces fixed
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit d1ddb0a6bb665dce10314bb5b9960e197e814820
Author: Matthias Beyer <mail@beyermatthias.de>
Date:   Sat Oct 5 18:02:10 2013 +0200

    drivers: usb: core: hcd: if-else-braces fixed

    Put else keyword on same line as closing brace from if statement, added
    { } braces as the styleguide says.

    Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/hcd.c b/drivers/usb/core/hcd.c
index 2a538ce..460bb59 100644
--- a/drivers/usb/core/hcd.c
+++ b/drivers/usb/core/hcd.c
@@ -879,9 +879,9 @@ static ssize_t authorized_default_store(struct device *dev,
  if (result == 1) {
   usb_hcd->authorized_default = val ? 1 : 0;
   result = size;
- }
- else
+ } else {
   result = -EINVAL;
+ }
  return result;
 }
 static DEVICE_ATTR_RW(authorized_default);
-- 
1.7.1